api/v1alpha1/envoygateway_types.go

```diff
@@ -503,11 +503,13 @@ type ExtensionManager struct {
 	// FailOpen defines if Envoy Gateway should ignore errors returned from the Extension Service hooks.
 	// The default is false, which means Envoy Gateway will fail closed if the Extension Service returns an error.
 	//
-	// Fail-close means that if the Extension Service hooks return an error, the relevant route/listener/resource
-	// will be replaced with a default configuration returning Internal Server Error (HTTP 500).
+	// Fail-close means that if the Extension Service hooks return an error, the error is propagated to the
```

Comment:
1. should we rename fail-close to fail-safe ?
2. instead of saying,  error is propagated to the xds translator, can we say - the xDS update specific the envoy proxy fleet is skipped and the previous good state is preserved

Reply:
> should we rename fail-close to fail-safe ?
fail safe can mean different things depending on the context. Maybe it's just better to not name the other failure state and just describe what it does. It's anyway not an enum in the API.

Reply:
Sure sgtm